hdf5-vol-daos.spec

```diff
@@ -32,7 +32,7 @@
 
 Name:    hdf5-vol-daos
 Version: 0.1
-Release: 3%{?relval}%{?dist}
+Release: 5%{?relval}%{?dist}
```

Comment:
Why is this jumping from 3 to 5?

Reply:
see other reply, a byproduct of some work in which it was thought that we might need an intermediate release. I will fix this to be Release 4.